gateway/src/main/java/org/georchestra/gateway/security/GatewaySecurityConfiguration.java

```diff
@@ -83,7 +81,7 @@ SecurityWebFilterChain securityWebFilterChain(ServerHttpSecurity http,
 
         // disable CSRF protection, considering it will be managed
         // by proxified webapps, not the gateway.
-        http.csrf().disable();
+        http.csrf().disable().exceptionHandling().accessDeniedHandler(new CustomAccessDeniedHandler());
```

Comment:
I'd separate from the ...disable() calls, because the modification has nothing to do with the comment abobe.